keyboards/index.php

```diff
@@ -66,6 +66,7 @@
   <form method='get' action='/keyboards' name='f'>
     <input id="search-q" type="text" placeholder="Enter language or keyboard" name="q" autofocus>
     <input id="search-f" type="image" src="<?= cdn('img/search-button.png"') ?>" value="Search" onclick="return do_search()">
+    <input id="search-obsolete" type="hidden" name="obsolete" value="0">
```

Comment:
Does the keyboard search in /_includes/2020/templates/Menu.php also get updated (phone-menu and top-menu1)?

Reply:
No, this is the only interface where obsolete keyboard searches will be possible; if the field is missing, then it's treated as `0`